x-pack/test/api_integration/apis/cases/privileges.ts

```diff
@@ -272,6 +371,86 @@ export default ({ getService }: FtrProviderContext): void => {
       });
     }
 
+    for (const { user, owner, userWithFullPerms } of [
+      { user: casesV3NoAssigneeUser, owner: CASES_APP_ID, userWithFullPerms: casesV3AllUser },
+      {
+        user: casesV2NoCreateCommentWithReopenUser,
+        owner: CASES_APP_ID,
+        userWithFullPerms: casesV3AllUser,
+      },
+      {
+        user: obsCasesV2NoCreateCommentWithReopenUser,
+        owner: OBSERVABILITY_APP_ID,
+        userWithFullPerms: obsCasesV3AllUser,
+      },
+      {
+        user: secCasesV2NoCreateCommentWithReopenUser,
+        owner: SECURITY_SOLUTION_APP_ID,
+        userWithFullPerms: secCasesV3AllUser,
+      },
+      { user: secReadUser, owner: SECURITY_SOLUTION_APP_ID, userWithFullPerms: secAllUser },
+      { user: casesOnlyDeleteUser, owner: CASES_APP_ID, userWithFullPerms: casesAllUser },
+      {
+        user: obsCasesOnlyDeleteUser,
+        owner: OBSERVABILITY_APP_ID,
+        userWithFullPerms: obsCasesAllUser,
+      },
```

Comment:
Nit: I think V3 variants of users: `casesNoDelete`,  `obsCasesNoDelete`, and `secAllCasesNoDelete` (the names are sort of misleading, the have the `minimal_all` privilege) should be added here.